lighthouse-core/audits/long-tasks.js

```diff
@@ -72,6 +77,7 @@ class LongTasks extends Audit {
     return {
       score: 1,
       details: tableDetails,
+      displayValue: str_(UIStrings.displayValue, {itemCount: results.length}),
```

Comment:
some audits dont set display value for zero items:

https://github.com/GoogleChrome/lighthouse/blob/f3d0e3459d8fd15b055148dec0ae4e430df6495b/lighthouse-core/audits/deprecations.js#L91

should we do the same here and the CLS audit?

Reply:
I've gone back and forth on it. I don't have a strong preference either way. For consistency, I can go ahead and do the same for these new audits.